src/main/java/dev/tycho/stonks/managers/DatabaseHelper.java

```diff
@@ -857,6 +858,246 @@ public void changeVerification(Player player, String companyName, boolean newVer
         }).execute();
   }
 
+  public void changeServiceMaxSubs(Player player, int maxSubs, Service service) {
+    Member m = service.getCompany().getMember(player);
+    if (m == null || !m.hasManagamentPermission()) {
+      player.sendMessage(ChatColor.RED + "You don't have management perms for this company");
+      return;
+    }
+
+    if (maxSubs > 0 && maxSubs < service.getNumSubscriptions()) {
+      player.sendMessage(ChatColor.RED + "You can't set the max subscriptions lower than the current number of subscriptions");
+      return;
+    }
+
+    service.setMaxSubscribers(maxSubs);
+    try {
+      databaseManager.getServiceDao().update(service);
+      player.sendMessage(ChatColor.GREEN + "Max subs updated to " + maxSubs + " :)");
+    } catch (SQLException e) {
+      e.printStackTrace();
+      player.sendMessage(ChatColor.RED + "SQL EXCEPTION TELL WHEEZY");
```

Comment:
Use `sendMessage(Player, String)` and don't use `ChatColor.RED` (Also follow the messages i've used for errors)